test/unit/sql/SqlService.js

```diff
@@ -173,44 +173,44 @@ describe('SqlServiceTest', function () {
         it('should construct a SqlResultImpl with default result type if it\'s not specified', async function () {
             const fake = sandbox.replace(SqlResultImpl, 'newResult', sandbox.fake(SqlResultImpl.newResult));
             await sqlService.execute('s', [], { cursorBufferSize: 1 });
-            fake.calledOnceWithExactly(
-                sqlService,
-                fakeSerializationService,
-                fakeConnection,
-                fakeQueryId,
-                1,
-                SqlServiceImpl.DEFAULT_FOR_RETURN_RAW_RESULT,
-                fakeClientUUID
+            fake.calledWith(
```

Comment:
Why do we use this instead?

Reply:
See https://github.com/hazelcast/hazelcast-nodejs-client/pull/1056#discussion_r716540978 and https://github.com/hazelcast/hazelcast-nodejs-client/pull/1056#discussion_r716541199